client/src/components/grants/Landing.tsx

```diff
@@ -1,28 +1,39 @@
-import { useSelector } from "react-redux";
-import Button from "../base/Button";
+import { useSelector, useDispatch } from "react-redux";
+import Button, { ButtonVariants } from "../base/Button";
 import { RootState } from "../../reducers";
+import { initializeWeb3 } from "../../actions/web3";
 
 function Landing() {
+  const dispatch = useDispatch();
   const props = useSelector((state: RootState) => ({
     web3Initialized: state.web3.initialized,
+    web3Error: state.web3.error,
   }));
 
+  const connectHandler = () => {
+    dispatch(initializeWeb3());
+  };
```

Comment:
@schultztimothy do we need to stop the normal link behaviour here?

```

  const connectHandler = (e) => {
    e.preventDefault();
    dispatch(initializeWeb3());
  };
  ```

Reply:
The default type for the button component is "button" which doesn't have a default event type. So we should be good 👍